lca-main.yaml

```diff
@@ -63,6 +63,14 @@ Parameters:
       Used only when a call audio source with Amazon Chime SDK Voice Connector or Demo Asterisk PBX Server is selected.
       Leave blank to automatically create new Amazon Chime SDK Voice Connector.
 
+  VoiceConnectorsRequireEncryption:
```

Comment:
I prefer not to expose this parameter in the main template, as it will clutter up the parameter list further, and will require README documentation which i don't see in this PR.   Let's include it only in the nested ChimeVC template, and you can always update just the nested stack.